pkg/ddc/alluxio/delete_volume_test.go

```diff
@@ -0,0 +1,235 @@
+package alluxio
+
+import (
+	"context"
+	datav1alpha1 "github.com/fluid-cloudnative/fluid/api/v1alpha1"
+	"github.com/fluid-cloudnative/fluid/pkg/ddc/base"
+	"github.com/go-logr/logr"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"reflect"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"testing"
+)
+
+type TestCase struct {
+	engine 			*AlluxioEngine
+	isDeleted 		bool
+	isErr 			bool
+}
+
+type NullLogger struct{}
+
+func (log NullLogger) Enabled() bool {
```

Comment:
I think you don't need define `type NullLogger struct`, just reuse the existing implementation.  Like https://github.com/fluid-cloudnative/fluid/blob/0cd802eb5562f400f0bdfb850342452ca3ad7452/pkg/ddc/jindo/transform_fuse_test.go#L105

Reply:
tks, LGTM